.circleci/config.yml

```diff
@@ -58,6 +58,9 @@ jobs:
 #      - run:
 #          name: Project Tests
 #          command: yarn test:projects:cra-ts
+      - run:
+          name: Bundler Tests
+          command: yarn test:bundlers:rollup
```

Comment:
would suggest `test:projects:rollup` - as this won't introduce additional entropy and will still suggest that stack that client is using for her project

Reply:
Moved into single entry, `yarn test:projects` 👍